docs/semgrep-code/pro-rules.md

```diff
@@ -8,95 +8,67 @@ tags:
+## Types of rules in the Semgrep Registry by author
 
-The Semgrep Registry includes the following kinds of rules:
 * **Community rules** - reviewed by the Semgrep team, these rules consist of contributions from Semgrep’s community. Community rules encompass a wide array of rules, including many that are made for security auditors.
 * **Third-party rules** - created directly by external contributors such as Trail of Bits, GitLab and many more.
 * **Private rules** - rules that can be authored and published by your own organization and for use only by your organization.
 * **Pro rules** - proprietary rules created by the Semgrep team targeted for security and software engineers who need accurate findings. These rules provide increased coverage for many programming languages and use the latest Semgrep features.
 
-## Introduction
-
-The goal of Pro rules is to provide a set of well-supported rules with improved coverage across languages and vulnerability types. Semgrep Pro rules are written using Semgrep’s latest features and, in general, target users who are looking to produce highly accurate, actionable findings.
-
 ## Semgrep Pro rules content
 
 Semgrep Pro rules provide improved coverage for many languages, including Java, JavaScript, TypeScript, Python, PHP, Ruby, C#, Swift, and Go.
 
-After using Semgrep Pro rules, you will see improved findings across many languages on specific classes of vulnerabilities (such as hard-coded secrets, injection vulnerabilities, deserialization, XXE, and many others) as well as increased support for frameworks and technologies (such as Express, Spring, Java Servlets, Laravel, Go net/http, React, Next.js, Angular).
+Semgrep Pro rules provide improved findings across many languages on specific classes of vulnerabilities, such as hard-coded secrets, injection vulnerabilities, deserialization, XXE, and many others, as well as increased support for frameworks and technologies such as Express, Spring, Java Servlets, Laravel, Go net/http, React, Next.js, and Angular.
 
 Semgrep's Security Research team plans to keep improving coverage by adding support for more languages and popular frameworks, as well as reducing potential false positives by monitoring rules’ performance.
 
-:::info
-See the [Rule updates](/release-notes/rule-updates) for an overview of updates and improvements released by Semgrep, Inc. for Semgrep’s rules, including Semgrep Pro rules.
-:::
-
-## Using Semgrep Pro rules
-
-You can use Semgrep Pro rules in both Semgrep AppSec Platform and your own CLI or CI environment.
+## Scan with Semgrep Pro rules
 
-* To use Pro Rules with Semgrep AppSec Platform, add them to your Policies page. The rules are then included in your next scan.
-* To use Pro Rules in your own CI or CLI environment, add the rules or rulesets through the `--config` flag.
+Your Semgrep AppSec Platform account already includes Pro rules, as they are included in the **Default** ruleset. These Pro rules run on all your scans.

```

Comment:
```suggestion
Your Semgrep AppSec Platform account already includes Pro rules that are likely to be widely useful, as they are included in the **Default** ruleset. These Pro rules run on all your scans.

```

Reply:
Is it worth mentioning that for them to run interfile, the interfile analysis toggle needs to be enabled also?